integration-tests/array-enum/pom.xml

```diff
@@ -46,6 +46,27 @@
     </plugins>
   </build>
   <profiles>
+    <profile>
+      <id>resteasy-classic</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-rest-client-jackson</artifactId>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>resteasy-reactive</id>
+      <dependencies>
+        <dependency>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
+        </dependency>
+      </dependencies>
+    </profile>
```

Comment:
I declared the profiles in sub-modules because each integration test has its own dependencies. If we declare the profiles in the `integration-tests` module, we'll need to declare the dependencies used by all tests.
But thinking better now, this might be a good price to pay in order to make it simpler. WDYT @ricardozanini ?

Reply:
Done.